### ISSUE TITLE
Transitions in nested resources

### ISSUE THREAD

**Issue by nkgm** [ember-animation/liquid-fire#121]
TL;DR
1. When transitioning to a nested resource, `fromRoute` and `toRoute` are passed the wrong information making it impossible to target the desired route transition.
2. In such a scenario where transitioning to a nested resource would entail rendering 2 or more nested liquid-outlets, the outer transition completes **before** all inner liquid-outlets have rendered.

Jsfiddle demonstrating the issues: http://emberjs.jsbin.com/wuyoxa/4

In this fiddle I'm using a "catch-all" transition for the purposes of logging what arguments are passed to the matchers, and also make sure a `fade` transition is always used regardless.

This is what my routes look like:
- Index
  - resource **users**
    - resource **user**
      - route **view**
      - route **edit**

To reproduce (1.) just navigate **Users->Mr Red**. You will see this output on your console:

```
fromRoute: [object Window] user.view
toRoute: user.view undefined 
```

To reproduce (2.) navigate again **Users->Mr Red** and notice how **only** "Back to Users" smoothly fades in, and then the nested `user/view` template pops up **with no animation**.

Can take a shot at producing the failing test cases if required.
